lib/components/common/CheckBox.tsx

```diff
@@ -1,27 +1,27 @@
-import { FormControlLabel } from "@mui/material";
```

Comment:
I ran the lint command and a couple of files got changes only related with linting. This is one of them